narwhals/_arrow/series.py

```diff
@@ -70,6 +118,53 @@ def __mul__(self, other: Any) -> Self:
         other = validate_column_comparand(other)
         return self._from_native_series(pc.multiply(self._native_series, other))
 
+    def __truediv__(self, other: Any) -> Self:
+        import contextlib
+
+        pa = get_pyarrow()
+        pc = get_pyarrow_compute()
+
+        ser = self._native_series
+        native_type = ser.type
+        original_type_is_int = pa.types.is_integer(
+            native_type
+        ) or pa.types.is_unsigned_integer(native_type)
+
+        if original_type_is_int:
+            ser = ser.cast(pa.float64())
+
+        other = validate_column_comparand(other)
+        result = pc.divide(ser, other)
+
+        if original_type_is_int:
+            with contextlib.suppress(pa.ArrowInvalid):
+                result = result.cast(native_type)
```

Comment:
TIL: this is same of

```py
try:
    result = result.cast(native_type)
except pa.ArrowInvalid:
    pass
```

Reply:
I'm not sure we should do this, perhaps we should follow Polars here?
```
In [1]: import polars as pl

In [2]: pl.Series([4, 6]) / pl.Series([2, 3])
Out[2]:
shape: (2,)
Series: '' [f64]
[
        2.0
        2.0
]
```

Reply:
Oh I see! Thanks for pointing that out! Adjusting it now!